source/blog/2019-05-11-rspec-rails-4-0-has-been-released.md

```diff
@@ -0,0 +1,181 @@
+* Adds support for JRuby on Rails 5.2 and 6
+* Add support for parameterised mailers (Ignatius Reza, #2125)
+* Add ActionMailbox spec helpers and test type (James Dabbs, #2119)
+* Add ActionCable spec helpers and test type (Vladimir Dementyev, #2113)
+* Add support for partial args when using `have_enqueued_mail`
+  (Ignatius Reza, #2118, #2125)
+* Add support for time arguments for `have_enqueued_job` (@alpaca-tc, #2157)
+* Improve path parsing in view specs render options. (John Hawthorn, #2115)
+* Add routing spec template as an option for generating controller specs.
+  (David Revelo, #2134)
+* Add argument matcher support to `have_enqueued_*` matchers. (Phil Pirozhkov, #2206)
+* Switch generated templates to use ruby 1.9 hash keys. (Tanbir Hasan, #2224)
+* Add `have_been_performed`/`have_performed_job`/`perform_job` ActiveJob
+  matchers (Isaac Seymour, #1785)
+* Default to generating request specs rather than controller specs when
+  generating a controller (Luka Lüdicke, #2222)
+* Allow `ActiveJob` matchers `#on_queue` modifier to take symbolic queue names. (Nils Sommer, #2283)
+* The scaffold generator now generates request specs in preference to controller specs.
+  (Luka Lüdicke, #2288)
+* Add configuration option to disable ActiveRecord. (Jon Rowe, Phil Pirozhkov, Hermann Mayer, #2266)
+*  Set `ActionDispatch::SystemTesting::Server.silence_puma = true` when running system specs.
+  (ta1kt0me, Benoit Tigeot, #2289)
+
+Bug Fixes:
+
+* `EmptyTemplateHandler.call` now needs to support an additional argument in
+  Rails 6. (Pavel Rosický, #2089)
+* Suppress warning from `SQLite3Adapter.represent_boolean_as_integer` which is
+  deprecated. (Pavel Rosický, #2092)
+* `ActionView::Template#formats` has been deprecated and replaced by
+  `ActionView::Template#format`(Seb Jacobs, #2100)
+* Replace `before_teardown` as well as `after_teardown` to ensure screenshots
+  are generated correctly. (Jon Rowe, #2164)
+* `ActionView::FixtureResolver#hash` has been renamed to `ActionView::FixtureResolver#data`.
+  (Penelope Phippen, #2076)
+* Prevent `driven_by(:selenium)` being called due to hook precedence.
+  (Takumi Shotoku, #2188)
+* Prevent a `WrongScopeError` being thrown during loading fixtures on Rails
+  6.1 development version. (Edouard Chin, #2215)
+* Fix Mocha mocking support with `should`. (Phil Pirozhkov, #2256)
+* Restore previous conditional check for setting `default_url_options` in feature
+  specs, prevents a `NoMethodError` in some scenarios. (Eugene Kenny, #2277)
+* Allow changing `ActiveJob::Base.queue_adapter` inside a system spec.
+  (Jonathan Rochkind, #2242)
+* `rails generate generator` command now creates related spec file (Joel Azemar, #2217)
+* Relax upper `capybara` version constraint to allow for Capybara 3.x (Phil Pirozhkov, #2281)
+* Clear ActionMailer test mailbox after each example (Benoit Tigeot, #2293)
+
+Breaking Changes:
+
+* Drops support for Rails below 5.0
+* Drops support for Ruby below 2.3
```

Comment:
I'm confused while this RFC is not a part of some markdown file in the root of the repo. I tend to think it's "below 2.2".

Reply:
Officially its now 2.2, we made a mistake about supported Ruby versions for Rails.